packages/deployment/Dockerfile.sdk

```
@@ -58,15 +73,32 @@ RUN \
   MODDABLE_URL="$MODDABLE_URL" \
   XSNAP_NATIVE_COMMIT_HASH="$XSNAP_NATIVE_COMMIT_HASH" \
   XSNAP_NATIVE_URL="$XSNAP_NATIVE_URL" \
-  bin/agd build
+  yarn build
 
 # Remove dev dependencies.
-RUN rm -rf packages/xsnap/moddable packages/xsnap/xsnap-native/build/tmp
-RUN mkdir golang/tmp && \
-  mv golang/cosmos/package.json golang/cosmos/build golang/cosmos/index.cjs golang/tmp/ && \
-  rm -rf golang/cosmos && mv golang/tmp golang/cosmos
-# FIXME: This causes bundling differences.  https://github.com/endojs/endo/issues/919
-# RUN yarn install --frozen-lockfile --production --network-timeout 100000
+WORKDIR /usr/src/agoric-sdk
+RUN rm -rf packages/xsnap/moddable xsnap-native/build/tmp
```

Is `xsnap-native` really located in `/usr/src/agoric-sdk/xsnap-native`?
```suggestion
RUN rm -rf packages/xsnap/moddable packages/xsnap/xsnap-native/build/tmp
```

---

thanks, part of that was under a different WORKDIR before